modules/scripts/account-check.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+get_enterprise_endpoint() {
+    default_endpoint="enterprise.cloud.ibm.com"
+    enterprise_endpoint="${IBMCLOUD_ENTERPRISE_API_ENDPOINT:-"$default_endpoint"}"
+    ENTERPRISE_ENDPOINT=${enterprise_endpoint#https://}
+}
+
+get_enterprise_endpoint
+
+read -r TF_INPUT
+ACCOUNT_ID=$(echo "$TF_INPUT" | jq -r '.account_id')
+IAM_TOKEN=$(echo "$TF_INPUT" | jq -r '.iam_token')
+
+URL="https://${ENTERPRISE_ENDPOINT}/v1/accounts/${ACCOUNT_ID}"
+
+HTTP_CODE=$(curl -s -o /tmp/account.json -w "%{http_code}" \
```

Comment:
instead of creating a temp file, can't the json just be stored as an environment variable?